### PR TITLE
Update installation.md

### PR DESCRIPTION
**Pull request by mohamedsharaf**
fix windows users path installation problem so laravel command work
 "laravel' is not recognized as an internal or external command"

### CODE REVIEW

File: installation.md [hunk 1]
```diff
@@ -35,7 +35,9 @@ First, download the Laravel installer using Composer:
 
     composer global require "laravel/installer=~1.1"
 
+Second step ,
```
Comment (GrahamCampbell) [inline installation.md]: no space before the comma please

File: installation.md [hunk 2]
```diff
@@ -35,7 +35,9 @@ First, download the Laravel installer using Composer:
...
 Make sure to place the `~/.composer/vendor/bin` directory in your PATH so the `laravel` executable can be located by your system.
+For windows users make sure to add `%USERPROFILE%\AppData\Roaming\Composer\vendor\bin` to your `path` Environment variable
```
Comment (GrahamCampbell) [inline installation.md]: missing fullstop

**Comment by mohamedsharaf** [comment]
removed and added the end of line stop point

File: installation.md [hunk 3]
```diff
@@ -35,8 +35,11 @@ First, download the Laravel installer using Composer:
 
     composer global require "laravel/installer=~1.1"
 
+Second,
```
Comment (GrahamCampbell) [inline installation.md]: you can't really follow a comma with a newline

Comment (GrahamCampbell) [inline installation.md]: in this context

**Comment by mohamedsharaf** [comment]
sorry missed that now fixed

File: installation.md [hunk 4]
```diff
@@ -35,7 +35,9 @@ First, download the Laravel installer using Composer:
 
     composer global require "laravel/installer=~1.1"
 
-Make sure to place the `~/.composer/vendor/bin` directory in your PATH so the `laravel` executable can be located by your system.
+Second, Make sure to place the `~/.composer/vendor/bin` directory in your PATH so the `laravel` executable can be located by your system.
```
Comment (GrahamCampbell) [inline installation.md]: "Secondly, make sure"

File: installation.md [hunk 5]
```diff
@@ -35,7 +35,9 @@ First, download the Laravel installer using Composer:
...
+
+For windows users make sure to add `%USERPROFILE%\AppData\Roaming\Composer\vendor\bin` to your `path` Environment variable.
```
Comment (GrahamCampbell) [inline installation.md]: Windows rather than windows

**Comment by mohamedsharaf** [comment]
done

File: installation.md [hunk 6]
```diff
@@ -35,7 +35,9 @@ First, download the Laravel installer using Composer:
 
     composer global require "laravel/installer=~1.1"
 
-Make sure to place the `~/.composer/vendor/bin` directory in your PATH so the `laravel` executable can be located by your system.
+Secondly, make sure to place the `~/.composer/vendor/bin` directory in your PATH so the `laravel` executable can be located by your system.
+
+For Windows  users make sure to add `%USERPROFILE%\AppData\Roaming\Composer\vendor\bin` to your `path` Environment variable.
```
Comment (GrahamCampbell) [inline installation.md]: shouldn't be 2 spaces after Windows, also, now you need a comma after the word "users"

Comment (GrahamCampbell) [inline installation.md]: Maybe drop the "For" too?

Comment (GrahamCampbell) [inline installation.md]: "Windows users, make sure to add"

**Comment by GrahamCampbell** [comment]
Please squash to one commit.

**Pull request closed without merging**